client/ayon_openrv/plugins/load/load_frames.py

```diff
@@ -22,9 +22,9 @@ class FramesLoader(load.LoaderPlugin):
     """Load frames into OpenRV"""
 
     label = "Load Frames"
-    families = ["*"]
-    representations = ["*"]
-    extensions = [ext.lstrip(".") for ext in IMAGE_EXTENSIONS]
+    product_types = {"*"}
+    representations = {"*"}
```

Comment:
This is still a `list` in ayon core currently: https://github.com/ynput/ayon-core/blob/b38f7132ca0c162f6d99274f3b6701b29723c632/client/ayon_core/pipeline/load/plugins.py#L27

```suggestion
    representations = ["*"]
```

Reply:
yes, but at the end are converte to `set` anyways.

Reply:
> yes, but at the end are converte to `set` anyways.

Where is that?

And in that case - it might be worth updating the `LoaderPlugin` in ayon core as well. My IDE will at least complain if I override attribute types on inherited classes (as it should!)

Reply:
https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/pipeline/load/plugins.py#L144C34-L144C52 . Will do PR in ayon-core 😄

Reply:
https://github.com/ynput/ayon-core/pull/386